### DISCUSSION TITLE
Rancher Desktop - Silent installation in Windows

### DISCUSSION THREAD

**Discussion by abcarcol** [rancher-sandbox/rancher-desktop#1857] [Q&A] [answered]
Dear developers,

Is it possible to install Rancher Desktop for Windows in "silent" mode? I tried the following without success:

.\Rancher.Desktop.Setup.1.1.1.exe /?
.\Rancher.Desktop.Setup.1.1.1.exe /s
.\Rancher.Desktop.Setup.1.1.1.exe /silent

**Comment by jandubois** [comment, marked as answer]
I don't think this is supported right now. I'll convert this discussion item into a feature request.

**Comment by vtmx** [comment]
I think Rancher use NSIS System:
https://silentinstallhq.com/nullsoft-scriptable-install-system-nsis-silent-install-how-to-guide

Examples:
```sh
# silent
setup.exe /S
# local folder
setup.exe /D="C:\Program Files\Rancher"
# all users and silent
setup.exe /ALLUSERS=1 /S
```

**Comment by blueboxes** [comment]
This command installs it silently successfully:

```
Invoke-WebRequest -Uri https://github.com/rancher-sandbox/rancher-desktop/releases/download/v1.8.1/Rancher.Desktop.Setup.1.8.1.msi -UseBasicParsing -OutFile .\rancher_setup.msi;
 
Start-Process msiexec.exe -Wait -ArgumentList '/I rancher_setup.msi /quiet' -Verb runAs;
```
